packages/app/src/app/overmind/namespaces/dashboard/actions.ts

```diff
@@ -2284,6 +2284,30 @@ export const importGitHubRepository = async (
   }
 };
 
+export const importGitHubRepositoryWithoutRedirect = async (
+  { state, effects }: Context,
+  { owner, name }: { owner: string; name: string }
```

Comment:
Could you change the parameters of `importGitHubRepository` in line 2255? This way, we don't duplicate the function, and defaulting it to `true` does not require any changes to other usages of that action.

```suggestion
  { owner, name, redirect = true }: { owner: string; name: string; redirect?: boolean }
```

Reply:
Great suggestion! I think I split it first because I wanted to return something else, but I can do it by using the redirect bool too.